### PR TITLE
feat(packages/sozluk-content): add new sozluk terms: big-data, iot, ux, dba

### PR DESCRIPTION
**Pull request by rasitds**

### CODE REVIEW

File: packages/sozluk-content/terms/big-data.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Big Data
+tags:
+  - data-analysis
+  - data-management
+  - data-processing
+---
+
+Büyük veri, işlenmesi ve analiz edilmesi zor olan çok büyük ve karmaşık veri kümelerini ifade eder.
```
Comment (usirin) [inline packages/sozluk-content/terms/big-data.mdx:9]: `zor` yerine `kompleks` kullansak daha iyi sanki?

File: packages/sozluk-content/terms/dba.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: DBA - Database Administrator
+tags:
```
Comment (cansirin) [inline packages/sozluk-content/terms/dba.mdx:3]: I think we should use kebab-case for the tags

eg: database-administration, database-management

Comment (rasitds) [reply packages/sozluk-content/terms/dba.mdx:3]: Oh, I totally forgot about that. Will send a commit to fix this, thanks!

**Review by usirin — APPROVED** [review]
💯

**Pull request merged**